bucket/foxit-pdf-reader.json

```diff
@@ -0,0 +1,28 @@
+{
+    "version": "11.0.1.49938",
+    "description": "View, annotate, form fill, and sign PDF across desktop, mobile, and web – no matter if you’re at the office, home, or on the go.",
+    "homepage": "https://www.foxit.com/",
+    "license": {
+        "identifier": "Freeware",
+        "url": "https://www.foxitsoftware.com/pdf-reader/eula.html"
```

Comment:
```suggestion
        "url": "https://www.foxit.com/pdf-editor/eula.html"
```

Reply:
It's weird that the Reader is forwarding to the Editor's EULA, but it seems to be true... Done